src/uct/ib/base/ib_iface.c

```diff
@@ -219,6 +219,11 @@ size_t uct_ib_address_size(uct_ib_iface_t *iface)
     if (IBV_PORT_IS_LINK_LAYER_ETHERNET(uct_ib_iface_port_attr(iface))) {
         return sizeof(uct_ib_address_t) +
                sizeof(union ibv_gid);  /* raw gid */
+    } else if (iface->is_global_addr) {
```

Comment:
maybe add && !iface->is_global_addr to next condition, to unite this with last case (else)?